vyper/signatures/interface.py

```diff
@@ -151,10 +151,8 @@ def extract_interface_str(code, contract_name, interface_codes=None):
     for idx, event in enumerate(events):
         if idx == 0:
             out += "# Events\n\n"
-        out += "{event_name}: event({{{args}}})\n".format(
-            event_name=event.name,
-            args=", ".join([arg.name + ": " + str(arg.typ) for arg in event.args])
-        )
+event_args_str = ', '.join([arg.name + ': ' + str(arg.typ) for arg in event.args])
```

Comment:
Looks like a spacing error leading to a logical change
```suggestion
        event_args_str = ', '.join([arg.name + ': ' + str(arg.typ) for arg in event.args])
```